corehq/apps/app_manager/fixtures/mobile_ucr.py

```diff
@@ -112,6 +111,11 @@ def _get_apps(self, restore_state, restore_user):
 
         if app_aware_sync_app:
             apps = [app_aware_sync_app]
+        elif toggles.RESTORE_ACCESSIBLE_REPORTS_ONLY and restore_state.params.is_webapps:
```

Comment:
This still needs a `and not request_user.can_edit_apps` or something, right?

Reply:
Yeah sorry just pushed the changes we had talked about. Turns out we need to use the `view_apps` permission as the indicator since a user who can view apps would be able to use live preview.